etc/disable-common.inc

```diff
@@ -172,3 +172,29 @@ blacklist ${PATH}/roxterm-config
 blacklist ${PATH}/terminix
 blacklist ${PATH}/urxvtc
 blacklist ${PATH}/urxvtcd
+
+# disable common suid programms
+blacklist ${PATH}/firejail
```

Comment:
Disabling firejail in jail is appropriate if you want stop firejailed (eg. symlinked to firejail) apps from being started inside of sandbox.
This is unsuitable for general usage, and will break a lot, if not most configurations.
Note that if you start firejailed app in existing sandbox it will run without any additional sandboxing features.